### PR TITLE
Updates to the wiki for ROS Jade

### PR DESCRIPTION
**Pull request by wjwwood**
@dirk-thomas @tfoote @esteve for review

@esteve can you merge and deploy before going home today?

### CODE REVIEW

File: macro/Version.py [hunk 1]
```diff
@@ -47,10 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'jade':
+        if version.lower() == 'kame':
```
Comment (esteve) [inline macro/Version.py]: Following the tradition of using the first word for codenames, shouldn't this be kinetic?

Comment (wjwwood) [inline macro/Version.py]: Uh, not sure, I didn't really think about it, but I guess that's right. Is that what you want it to be?

Comment (esteve) [inline macro/Version.py]: Let's use kinetic, though I really like kame more because it'll translate to turtle turtle in Japanese :-)

**Comment by wjwwood** [comment]
Ok, I updated the name, I'll update the other pr's and wiki pages too.

**Comment by dirk-thomas** [comment]
lgtm

**Comment by esteve** [comment]
@wjwwood is this ready to merge or need to be merged at the same time as ros-infrastructure/rosbrowse#3 ?

**Comment by wjwwood** [comment]
It's good to go, we need to deploy rosbrowse as well, but they don't have to happen all at exactly the same time.

**Comment by esteve** [comment]
Deployed.

**Pull request merged**